tests/tools/FFI/bluechi-tester/bluechi-tester.go

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/spf13/cobra"
+)
+
+// service, object and register interface for basic client connection
+const (
+	BcDusInterface = "org.eclipse.bluechi"
+	BcObjectPath   = "/org/eclipse/bluechi/internal"
+	MethodRegister = "org.eclipse.bluechi.internal.Manager.Register"
+)
+
+// object path and signals of internal agents interface
+// see https://github.com/containers/bluechi/blob/main/data/org.eclipse.bluechi.internal.Agent.xml#L81-L116
+const (
+	InternalAgentObject = "/org/eclipse/bluechi/internal/agent"
+	InternalAgentSignal = "org.eclipse.bluechi.internal.Agent."
```

Comment:
```suggestion
	InternalAgentSignalPrefix = "org.eclipse.bluechi.internal.Agent."
```
What do you think? @dougsland